doc/wallet-backup.md

```diff
@@ -0,0 +1,85 @@
+
+There are multiple ways to make sure you have at least one other copy of the private keys needed to spend ZEC and view shielded ZEC.
+
+For all methods, you will need to include an export directory setting in your config file (`zcash.conf` located in the data directory which is `~/.zcash/` unless it's been overridden with `datadir=` setting):
+
+`exportdir=/path/to/chosen/export/directory`
+
+You may chose any directory as the location for export & backup files. If the directory doesn't exist, it will be created.
+
+### Using `backupwallet`
+
+To create a backup of your wallet, use:
+
+`zcash-cli backupwallet <nameofbackup>`.
+
+The backup will be an exact copy of the current state of your wallet.dat file stored in the export directory you specified in the config file. The file path will also be returned.
+
+If you generate a new Zcash address, it will not be reflected in the backup file.
+
+If your original `wallet.dat` file becomes inaccessible for whatever reason, you can use your backup by placing it into your data directory and renaming it to `wallet.dat`.
+
+### Using `z_exportwallet` & `z_importwallet`
+
+If you prefer to have an export of your private keys in human readable format, you can use:
+
+`zcash-cli z_exportwallet <nameofbackup>`
+
+This will generate a file in the export directory listing all transparent and shielded private keys with their associated public addresses. The file path will be returned in the command line.
+
+To import keys into a wallet which were previously exported to a file, use:
+
+`zcash-cli z_importwallet <nameofbackup>`
+
+### Using `z_exportkey`, `z_importkey`, `dumpprivkey` & `importprivkey`
+
+If you prefer to export a single private key for a shielded address, you can use:
+
+`zcash-cli z_exportkey <z-address>`
+
+This will return the private key and will not create a new file.
+
+For exporting a single private key for a transparent address, you can use the command inherited from bitcoin:
+
+`zcash-cli dumpprivkey <t-address>`
+
+This will return the private key and will not create a new file.
+
+To import a private key for a shielded address, use:
+
+`zcash-cli z_importkey <z-priv-key>`
+
+This will add the key to your wallet and rescan the wallet for associated transactions if it is not already part of the wallet.
+
+See the command's help documentation for instructions on fine-tuning the wallet rescan:
+
+`zcash-cli help z_importkey`
+
+To import a private key for a transparent address, use:
+
+`zcash-cli importprivkey <t-priv-key>`
+
+This has the same functionality as `z_importkey` but works with transparent addresses.
+
+See the command's help documentation for instructions on fine-tuning the wallet rescan:
+
+`zcash-cli help importprivkey`
+
+### Using `dumpwallet`
+
+This command inherited from bitcoin is depreciated. It will export private keys in a similar fashion as `z_exportwallet` but only for transparent addresses.
```

Comment:
dumpwallet behavior is confusing.
The command exists but I couldn't get it to work - perhaps because of this deprecation.
It might be better to delete the command if it doesn't work.